indexer-app/test/cmr/indexer/test/data/index_set_generics.clj

```diff
@@ -5,8 +5,8 @@
 
 (def index-definition-def
   {:MetadataSpecification
-   {:URL "https://cdn.earthdata.nasa.gov/generics/dataqualitysummary/v1.0.0"
-    :Name "DataQuailtySummary",
+   {:URL "https://cdn.earthdata.nasa.gov/generics/data-quality-summary/v1.0.0"
+    :Name "DataQualitySummary",
```

Comment:
Did Erich weigh in on adding spaces here?

Reply:
Actually, I think only the public interface is required to add the dashes...

Reply:
I think (someone correct me if I'm wrong) that while the dashes were only required of the public one, we opted to change both because they will be copies of each other.

Reply:
It's safe to change it all as you don't know when the two parts are interacted.

Reply:
Done - fixed :Name to match